python/cuml/test/test_random_forest.py

```diff
@@ -217,10 +221,14 @@ def test_rf_classification(small_clf, datatype, split_algo,
      (1, 'sqrt', False, 100),
      (1, 1.0, True, 17),
      (1, 1.0, True, 32),
+     (0, 1.0, True, 16),
+     (1, 1.0, True, 11),
+     (0, 'auto', True, 128),
```

Comment:
Fantastic news on solving the bug @venkywonka (and @vinaydes) I think we should just add some test cases like n_bins 100 back to have better test coverage and we’re good to merge

Reply:
yep will add them right away!